openshift/README.md

```diff
@@ -6,6 +6,20 @@ Red Hat Data Grid (RHDG) for OpenShift
 **Target Product:** Red Hat Data Grid  
 **Product Versions:** RHDG 7.3 or later
 
+Available Quickstarts
+---------------------
+Data Grid for OpenShift quickstart tutorials:
+
+| **Quickstart Name** | **Shows you how to...** |
+|:-----------|:-----------|
+| [Hello World](shared/hello-world/README.md) | Quickly verify `cache-service` and `datagrid-service` deployments. |
+| [External Access](shared/external-access/README.md) | Access `cache-service` and `datagrid-service` pods outside OpenShift. |
+| [Monitoring](shared/prometheus-monitoring/README.md) | Monitor Data Grid for OpenShift with Prometheus and Grafana. |
+| [Cache Service: Creating Caches](cache-service/create-cache/README.md) | Connect to `cache-service` over Hot Rod and create permanent caches. |
+| [Data Grid Service: Creating Caches](datagrid-service/create-cache/README.md) | Connect to `datagrid-service` over Hot Rod and create permanent caches. |
```

Comment:
This might be a good opportunity to quickly the different between the `create-cache` tutorials in cache and datagrid service. With the former only allowing new caches that are copies of the default cache and the latter allowing any type of cache to be created. WDYT?